README.md

```diff
@@ -113,12 +115,34 @@ catkin do the following:
       ## TeXlipse
       .texlipse
       ```
+
+#### Multiple Packages
+The _eclipsify_ command can be used in conbitation with the catkin command to easily generate project files for all the ROS packages that  
+reside in a workspace. The options are as follows:
+
+  1. Create project files in the catkin worspace devel directory:
+    ```
+    catkin list -u | xargs -I pkg bash -c "eclipsify pkg"
+    ```
+    As in the single package example, the eclipse project files will be stored in a subdirectory in the **\<catkin workspace>/devel/share/..**  
+    directory.
+
+  2. Create project files in a separate __projects__ directory:
+    Run the following command from the root directory of your catkin workspace
+    ```
+    catkin list -u | xargs -I pkg bash -c "eclipsify pkg -O projects/pkg"
+    ```
+    This will create the eclipse project files for each package in separate subdirectories in the **\<catkin workspace>/projects/** directory.
+
+
 - **Import** into an Eclipse workspace through one of the following alternatives:
 
   1. Import the project by browsing to the project files directory just created from the **Import** window in eclipse.
   Make sure to select the "Existing Project into Workspace" option under the "General" category
   shown in the **Import** window.
   
-  2. **EGit** option (assuming my_pkg being under git version control):
+  2. In the next window, check the boxes for all the packages you wish to import and then click **Finish**
```

Comment:
Careful, this should not be another _alternative_ (see above). It should be another paragraph below `1.` or there should be sub bullet points.